challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>Is this data on /project/informatics_lab/ ?</p><p>If not, we should make a copy there, because I think its good to have in a shared location that is less likely to be accidentally or on purpose deleted when cyril runs out of disk space.</p><p>I can do the copying for you if you don't have permissions just tell me what needs to be copied.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I have copied over the data, no issues on permissions